src/components/repl/ReplToolbar.tsx

```diff
@@ -55,6 +62,25 @@ const ReplToolbar: FC<Props> = ({
   const isRunning = [PyodideStatus.RUNNING, PyodideStatus.WAIT_INPUT].includes(
     status,
   );
+  const [isFullScreen, setIsFullScreen] = useState(false);
+
+  const onClickFullScreen = (): void => {
+    const bodyElem = document.body;
+    if (!document.fullscreenElement) {
+      bodyElem
+        .requestFullscreen()
+        .then(() => setIsFullScreen(true))
+        .catch((err) => {
+          // eslint-disable-next-line no-console
+          console.log(
+            `Error attempting to enable fullscreen mode: ${err.message} (${err.name})`,
+          );
+        });
+    } else {
+      document.exitFullscreen();
+      setIsFullScreen(false);
+    }
+  };
```

Comment:
I've recently read that it's really useful to have custom hook, I guess that it is a good example! If we want to add it to many apps, it could be exported from graasp-sdk for example. (you don't have to do it now)